configs/terraform/environments/prod/modules-submission.tf

```
@@ -46,6 +46,13 @@ resource "google_artifact_registry_repository_iam_member" "dev_modules_internal_
   member     = "serviceAccount:${google_service_account.kyma_project_kyma_submission_pipeline.email}"
 }
 
+resource "google_artifact_registry_repository_iam_member" "kyma_modules_admin_sre" {
+  provider   = google.kyma_project
+  repository = var.kyma_modules_repository.name
+  role       = "roles/artifactregistry.admin"
```

```suggestion
  role       = "roles/artifactregistry.admin"
```
I think it should be the reader role instead of the admin.